test/client.test.js

```diff
@@ -3,7 +3,7 @@
 const util = require('util')
 const assert = require('assert')
 const tap = require('tap')
-const uuid = require('uuid')
+const { v4: uuidv4 } = require('uuid')
```

Comment:
I think we can probably replace this dependency completely with code like the following:

```js
const crypto = require('crypto')
const bindDN = crypto.randomBytes(16).toString('hex')
```